examples/basics/demo_spc.py

```diff
@@ -0,0 +1,223 @@
+r"""
+Tour of Ordering Algorithms in Single Pixel Camera
```

Comment:
```suggestion
Pattern Ordering in a Compressive Single Pixel Camera
```
we should remove the name tour I believe - we have too many 'tours' already :/

Reply:
okey, thanks!